src/components/CaseDetails.vue

```diff
@@ -0,0 +1,368 @@
+<template>
```

Comment:
what is this file for ?
remove the file if this is not related to  the case details page.

Reply:
removed file